mysql-test/suite/innodb/t/doublewrite.opt

```diff
@@ -0,0 +1 @@
+--innodb_flush_sync=0
```

Comment:
This parameter will make the test extremely slow. When I ran 20 concurrent instances of the test, only about 3.2 execution cores were occupied on the average. The test execution time would be 25 to 27 seconds. OK, I see that on https://buildbot.mariadb.net/buildbot/builders/fulltest-debug-big-only/builds/13772 (10.4) the execution time is similar, but I think that my system is generally much faster than that builder.

Reply:
If we really need this parameter, can we please use `SET GLOBAL innodb_flush_sync=0` at the exact places where it is needed?